tests/test_twitter_backfill.py

```
@@ -0,0 +1,308 @@
+            "createdAt": _years_ago_iso(10),
+            "isBlueVerified": True,
+            "isVerified": True,
+        })
+        assert b["followers"] == 25
+        assert b["age"] == 10
+        assert b["blue_verified"] == 5
+        assert b["legacy_verified"] == 10
+        assert score == 50
+
+    def test_company_affiliation_adds_10(self):
+        score, b = compute_partial_author_weight({
```

Ruff (0.15.12)

[warning] Unpacked variable `score` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)